Source/VCGeneration/VCGen.cs

```diff
@@ -637,6 +637,10 @@ public VCExpr CodeExprToVerificationCondition(CodeExpr codeExpr, List<VCExprLetB
       }
 
       CumulativeAssertionCount += assertionCount;
+      if (assertionCount == 0)
+      {
+        return VCExpressionGenerator.True;
```

Comment:
So, after `True` is returned, what code prevents this trivial VC from being sent to the SMT solver?

Reply:
The VC will be sent to the SMT solver.   My PR description may be misleading.  I only wanted to claim that the original VC is not sent to the SMT solver.  But a trivial VC will be sent.

Reply:
@RustanLeino : Does this fix look reasonable to you?